### DISCUSSION TITLE
Update and build libmesh failing on HPC cluster

### DISCUSSION THREAD

**Discussion by ekerrison** [idaholab/moose#19586] [Q&A Getting Started] [answered]
Hi there,

I'm trying to install MOOSE on a cluster for local use (not cluster wide). I've followed the instructions on the website ([https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html](url)) except for the fact that the HPC already has PETSc installed, so I just load the module in using `module load cray-petsc`. I also load in `cray-mpich` and `cray-tpsl`.

Libmesh fails to build with the following message:

`Makefile:28457: recipe for target 'src/systems/libmesh_opt_la-system_projection.lo' failed
make[1]: *** [src/systems/libmesh_opt_la-system_projection.lo] Error 1
make[1]: Leaving directory '/group/project/software/moose/libmesh/build'
Makefile:32536: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1`

This happens using gcc 6.1.0 and gcc 7.2.0 (in case that is helpful). Hopefully someone here has an idea of what's going wrong, I'm totally stumped!

**Comment by GiudGiud** [comment, marked as answer]
Hello

There should be a build log around, a file named `config.log`. Could you please upload it here

Guillaume

**Reply by GiudGiud** [reply]
Hello
Any update on this?
